Pipfile

```diff
@@ -29,4 +29,4 @@ six = "==1.12.0"
 Werkzeug = "==0.15.3"
 
 [requires]
-python_version = "3.8"
+python_version = "3.9"
```

Comment:
if you update the python version i think you'll need to update these too https://github.com/uc-cdis/tube/blob/677b739/.travis.yml#L5-L11
and the Dockerfile base image